presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/PlanChecker.java

```diff
@@ -58,6 +58,7 @@ public PlanChecker(FeaturesConfig featuresConfig, boolean forceSingleNode)
                         new NoIdentifierLeftChecker(),
                         new VerifyNoFilteredAggregations(),
                         new VerifyNoOriginalExpression(),
+                        new NoSubqueryRowExpressionLeftChecker(),
```

Comment:
perhaps call this VerifyNoSubqueryRowExpression similar to the one above
(I find "left" a bit confusing because of the other meaning left/right)